init_files/boot_scripts/hello_world.sh

```diff
@@ -1,10 +0,0 @@
-#!/bin/bash
-
-################################################################
-#
-#  This Script can say 'Hello World'
-#  
-#  hello_world.sh -- Version 1.0
-################################################################
-
-echo 'Hello World. This is an example of a boot script!'
```

Comment:
I dunno, they are literally just "hello world" in bash and Python. Is the intended audience for the "boot scripts" feature (i.e. users) likely to look in this repo?